cmd/burrow/commands/start.go

```diff
@@ -36,16 +33,34 @@ func Start(output Output) func(cmd *cli.Cmd) {
 
 			output.Logf("Using validator address: %s", *conf.ValidatorAddress)
 
-			ctx, cancel := context.WithCancel(context.Background())
-			defer cancel()
-
-			kern, err := conf.Kernel(ctx, *restoreDumpOpt)
+			kern, err := core.NewKernel()
 			if err != nil {
 				output.Fatalf("could not create Burrow kernel: %v", err)
 			}
 
-			err = kern.Boot()
-			if err != nil {
+			if err = kern.ExecutorFromConfig(conf.Execution); err != nil {
```

Comment:
Hm okay I see how the names `FromConfig` line up now. I feel like these all ought to be `LoadXFromConfig` though it is more long-winded since they do return the thing they load. Alternatively `LoadX` but I think i prefer `LoadKeysFromConfig` `LoadLoggerFromConfig`

Reply:
Yeah I agree